status/projects.py

```diff
@@ -754,6 +754,8 @@ def make_project_running_note(application, project, note, category, user, email)
     def notify_tagged_user(application, userTags, project, note, category, tagger, timestamp):
         view_result = {}
         time_in_format = datetime.datetime.strptime(timestamp, '%Y-%m-%d %H:%M:%S').strftime("%a %b %d %Y, %I:%M:%S %p")
+        note_id = 'running_note_' + project + '_' + \
+                    str(int((datetime.datetime.strptime(timestamp, '%Y-%m-%d %H:%M:%S') -  datetime.datetime.utcfromtimestamp(0)).total_seconds()))
```

Comment:
I don't understand the ` -  datetime.datetime.utcfromtimestamp(0)).total_seconds()` part...

Reply:
But I guess it's needed and it works so I can live with it.